scripts/set-environment

```diff
@@ -19,7 +19,8 @@ set -eo pipefail
 #
 CI_BRANCH=${CI_BRANCH:-$CIRCLE_BRANCH}
 CI_BUILD_NUMBER=${CI_BUILD_NUMBER:-$CIRCLE_BUILD_NUM}
-CI_PROJECT_NAME=${CI_PROJECT_NAME:-CIRCLE_PROJECT_REPONAME}
+CI_PROJECT_NAME=${CI_PROJECT_NAME:-$CIRCLE_PROJECT_REPONAME}
+CI_PULL_REQUEST=${CI_PULL_REQUEST:-$CI_PULL_REQUEST}
```

Comment:
Hi,

I think this should be:

```
CI_PULL_REQUEST=${CI_PULL_REQUEST:-$CIRCLE_PULL_REQUEST}
```

Could you please make this change? Thanks!

Reply:
GOSH thank you for pointing it out @kporras07